scripts/build_p2pd.sh

```diff
@@ -70,10 +91,12 @@ build_target() {
 		FAILED=0
 		go get ./... && break || FAILED=1
 		CURR=$(( CURR + 1 ))
-		echo "retry #${CURR}"
+        if $verbose; then
+            echo "retry #${CURR}"
+        fi
 	done
 	if [[ $FAILED == 1 ]]; then
-		echo "Error: still fails after retrying ${MAX_RETRIES} times."
+        echo "Error: still fails after retrying ${MAX_RETRIES} times."
```

Comment:
Is it missing a tab?  it seems unaligned with the exit on next line

Reply:
This is weird, it looks ok on my editor